Backend/Database/DatabaseContext.cs

```diff
@@ -77,7 +77,7 @@ internal List<ProjectResponse> GetProjectsByFilter(string[]? skills, string[]? i
         return (DbErrorStatusCodes.Ok, projects);
     }
 
-    internal bool PopulateProjects(int count = 10)
+    internal bool PopulateProjects(int count = 1000)
```

Comment:
1000 might be overkill, maybe 100 would be better or sth

Reply:
Oscar wished for 1000 and yes 100 is enough also